crates/prover/src/runner.rs

```diff
@@ -650,3 +894,32 @@ async fn get_initial_slot_height<Da: DaSpec>(client: &SequencerClient) -> u64 {
         }
     }
 }
+
+async fn get_da_block_at_height<Da: DaService>(
```

Comment:
I believe that this function is also declared in `fullnode/runner.rs`.

What i wanted to do with this is move it to `crates/primitives` where shared functionality between sequencer / fullnode / prover would be placed.